src/Storages/StorageURL.cpp

```diff
@@ -153,6 +155,26 @@ namespace
     {
         return parseRemoteDescription(uri, 0, uri.size(), '|', max_addresses);
     }
+
+    auto proxyConfigurationToPocoProxyConfiguration(const ProxyConfiguration & proxy_configuration)
+    {
+        Poco::Net::HTTPClientSession::ProxyConfig poco_proxy_config;
+
+        poco_proxy_config.host = proxy_configuration.host;
+        poco_proxy_config.port = proxy_configuration.port;
+        poco_proxy_config.protocol = ProxyConfiguration::protocolToString(proxy_configuration.protocol);
```

Comment:
Is there any way to get "any" here and how wold that behave with that value?

Reply:
Yes and no. The function `ProxyConfiguration::protocolToString` can indeed return `"any"`, but this code path will never lead to that.

That's because this os capturing the return value of the resolver. A resolver will return either `http` or `https`, protocol `any` does not exist :P.